packages/phishing-controller/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [12.3.0]
+
+### Fixed
+
+- Fixed extension performance issues ([#4853](https://github.com/MetaMask/core/pull/4853))
```

Comment:
Would it be worth going into more detail about what sort of issues are being fixed? It saves a bit of time when reviewing changelogs if we don't have to look back at the PR to understand changes.